project-reports/2023/2023-Q2-Hyperledger-Iroha.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: default
+title: 2023 Q2 Hyperledger Iroha
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+
+Currently, active maintenance of Iroha 1 has stopped and there are no plans for major releases. However, discussions to release the current RC continue because it will require more than one team switching from everyday tasks to focus on this.
+
+All Iroha 2 developers are now fully onboarded and at maximum productivity.
+
+Since development of the Ursa project has been discontinued, the Iroha team is rewriting its essential parts to integrate in the Iroha codebase. In addition, thanks to Arun S M, the Iroha 2 BCE team will start collaborating with the Hyperledger Blockchain Explorer team.
```

Comment:
```suggestion
Since development of the Ursa project has been discontinued, the Iroha team is rewriting its essential parts to integrate in the Iroha codebase. In addition, thanks to Arun S M, the Iroha 2 BCE team will start collaborating with the Blockchain Explorer, a Hyperledger Lab, team.
```
We can no longer refer to this lab as Hyperledger Explorer.

Reply:
Thanks, I will remember that!